### PR TITLE
Results Page

### PR DESCRIPTION
**Pull request by alexanderallen-aa**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

### CODE REVIEW

File: packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  ExpoJudgingSessionResult,
+  fetchExpoJudgingSessionResults,
+} from '../../../../pageUtils/expoJudgingSession/[id]/fetchExpoJudgingSessionResults';
+import { openErrorToast } from '../../../../components/utils/CustomToast';
+
+const ExpoJudgingSessionDetails: NextPage = () => {
+  const { expoJudgingSession } = useExpoJudgingSessionFetcher();
+  const [expoJudgingSessionResults, setExpoJudgingSessionResults] =
+    React.useState<ExpoJudgingSessionResult[]>();
+  React.useEffect(() => {
```
Comment (SpencerKaiser) [inline packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx:16]:
```suggestion

  React.useEffect(() => {
```

File: packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          return;
+        }
+        if (Array.isArray(results)) {
+          if (results) {
+            if (results.length === 0) {
+              openErrorToast({ title: 'No results found' });
+            }
+            setExpoJudgingSessionResults(results);
+          }
+        }
```
Comment (SpencerKaiser) [inline packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx:43]: The type checking here should be exhaustive so the only other potential type for this is an array so these should be unnecessary
```suggestion


        if (results.length === 0) {
          openErrorToast({ title: 'No results found' });
        }
        setExpoJudgingSessionResults(results);

```

File: packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    };
+    if (expoJudgingSession) {
+      void getExpoJudgingSessionResults();
+    }
+  }, [expoJudgingSession]);
+
+  return (
+    <PageContainer
+      pageTitle={'Expo Judging'}
+      heading={'Expo Judging'}
```
Comment (SpencerKaiser) [inline packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx:54]: I think this is a better title; thoughts?
```suggestion
      pageTitle={'Expo Judging Overview'}
      heading={'Expo Judging Overview'}
```

**Review by SpencerKaiser — CHANGES_REQUESTED** [review]
Small tweaks and then good to go

File: packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx (hunk 1 above)
Comment (alexanderallen-aa) [reply packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx:16]: that's not me, that is prettier.

Comment (SpencerKaiser) [reply packages/web/src/pages/admin/expoJudgingSession/[id]/index.tsx:17]: I merged the commit from the UI and Prettier is fine with it

**Review by SpencerKaiser — APPROVED** [review]

**Pull request merged**